xcube_sh/store.py

```diff
@@ -306,30 +306,33 @@ def get_data_store_params_schema(cls) -> JsonObjectSchema:
         )
 
     @classmethod
-    def get_type_ids(cls) -> Tuple[str, ...]:
-        return TYPE_ID_DATASET,
+    def get_type_specifiers(cls) -> Tuple[str, ...]:
+        return str(TYPE_SPECIFIER_CUBE),
 
-    def get_data_ids(self, type_id: str = None) -> Iterator[Tuple[str, Optional[str]]]:
-        self._assert_valid_type_id(type_id)
-        metadata = SentinelHubMetadata()
-        for data_id, dataset in metadata.datasets.items():
-            yield data_id, dataset.get('title')
+    def get_type_specifiers_for_data(self, data_id: str) -> Tuple[str, ...]:
```

Comment:
Should this method check that the supplied `data_id` is valid?

Reply:
Yes.